client/client.go

```diff
@@ -316,9 +316,10 @@ func (c *Client) useInsecureHTTPClient(insecure bool) *http.Transport {
 }
 
 // Takes raw payload and does the http request
-//  Used for login request
-//  passwords with special chars have issues when using container
-//  for encoding/decoding
+//
+//	Used for login request
+//	passwords with special chars have issues when using container
+//	for encoding/decoding
```

Comment:
If these are two different sentences, can it start with capital letter and with a full stop?